<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>755</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>102</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Chekiang Sung</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,4,1,4,1,2">
      <item>
       <layout class="QGridLayout" name="Grid_Title">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="Title">
          <property name="font">
           <font>
            <family>Chekiang Sung</family>
            <pointsize>60</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>樂來樂生氣銀行</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="Grid_Detail" columnstretch="1,1,1,2">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>本行利率</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QGridLayout" name="Grid_Length" rowstretch="1" columnstretch="1,1,1">
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>分鐘</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>存期</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="LCD_Length">
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>30</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="smallDecimalPoint">
             <bool>true</bool>
            </property>
            <property name="digitCount">
             <number>1</number>
            </property>
            <property name="value" stdset="0">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QGridLayout" name="Grid_Rate" rowstretch="0" columnstretch="1,1,1">
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>利率</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLCDNumber" name="LCD_Interest_Rate">
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>50</pointsize>
             </font>
            </property>
            <property name="digitCount">
             <number>1</number>
            </property>
            <property name="value" stdset="0">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>30</pointsize>
             </font>
            </property>
            <property name="text">
             <string>%</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="Grid_Time" columnstretch="1,1,3">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_Time">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>現在時間</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="LCD_Time">
          <property name="digitCount">
           <number>8</number>
          </property>
          <property name="value" stdset="0">
           <double>11022033.000000000000000</double>
          </property>
          <property name="intValue" stdset="0">
           <number>11022033</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="Grid_Group" columnstretch="1,1,3">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_Group">
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>帳號</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="LE_Group">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_Serve">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>10</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>10</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>127</red>
                <green>127</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>請選擇服務項目</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_ServeEN">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>10</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>255</green>
                <blue>10</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>127</red>
                <green>127</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>PLEASE SELECT A TRANSACTION</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="Grid_Button" columnstretch="4,0,0,0,0">
        <item row="0" column="1">
         <spacer name="HSA1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="BT_AccountSearch">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>餘額查詢 INQUIRY</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer name="HSA2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="BT_WithDrawal">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>帳戶提款 WITHDRAWAL</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="BT_Deposit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>帳戶存款 DEPOSIT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="HS2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QGridLayout" name="Grid_Slogan">
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="L_Slogan">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>20</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>一 時 存 錢 一 時 爽 ， 一 直 存 錢 一 直 爽 ， 錢 多 就 比 誰 更 浪 ， 身 價 翻 倍 紅 不 讓</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
